Sources/MapboxNavigation/DayStyle.swift

```diff
@@ -262,10 +262,6 @@ open class DayStyle: Style {
             LaneView.appearance(for: phoneTraitCollection).secondaryColor = .defaultLaneArrowSecondary
             LaneView.appearance(for: phoneTraitCollection).primaryColorHighlighted = .defaultLaneArrowPrimaryHighlighted
             LaneView.appearance(for: phoneTraitCollection).secondaryColorHighlighted = .defaultLaneArrowSecondaryHighlighted
-            LaneView.appearance(for: phoneTraitCollection, whenContainedInInstancesOf: [LanesView.self]).primaryColor = .defaultLaneArrowPrimary
```

Comment:
There is no need to set appearance for cases when `LaneView` is inside `LanesView`, because setting appearance for `phoneTraitCollection` covers this case.